docs/tutorials/getting-started-with-kev.md

```diff
@@ -139,23 +139,23 @@ Kev has now been initialised and configured. It has,
 - Created `dev` (a sandbox used by Kev for continuous development), `local` (useful for testing on our own machine) and `staging` (useful for testing on a remote machine) _Compose environment overrides_.
 
 It has also generated four files:
-- `kev.yaml`, a manifest that describes our _source application definition_ and _Compose environment overrides_.
-- Three `docker-compose.kev.*.yaml` files to represent our _Compose environment overrides_.
+- `appmeta.yaml`, a manifest that describes our _source application definition_ and _Compose environment overrides_.
+- Three `docker-compose.env.*.yaml` files to represent our _Compose environment overrides_.
 
-#### Manifest: kev.yaml
+#### Manifest: appmeta.yaml
```

Comment:
Could we rephrase the tile of this section?

Reply:
Hm. If we rephrase this, we should also rephrase the following section header, which states the override files. Suggest maybe `Application metadata file` and `Compose environment override files` (and drop the filenames from the section headings) - then I'll reference the filenames in the section text